src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,182 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.InvalidUserAgeException;
+import core.basesyntax.exception.InvalidUserLoginException;
+import core.basesyntax.exception.InvalidUserPasswordException;
+import core.basesyntax.exception.UserAlreadyExistsException;
+import core.basesyntax.exception.UserValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+
+    private RegistrationService registrationService = new RegistrationServiceImpl();
+    private User validUser = new User();
```

Comment:
```suggestion
    private RegistrationService registrationService
    private User validUser
```

Reply:
I already had such version of the code, but another mentor suggested doing it as it is now

Reply:
In tests this is not that big of the deal, either way it will be created only once, that is more of a style question where we want to do this, I personally think BeforeEach is a more traditional approach, like initializing objects in constructor, but field is fine too